packages/esm-patient-orders-app/src/edit-lab-results/modal/edit-lab-results.modal.tsx

```diff
@@ -0,0 +1,118 @@
+import { Button, ModalBody, ModalFooter, ModalHeader, RadioButton, RadioButtonGroup } from '@carbon/react';
+import { launchWorkspace, useSession } from '@openmrs/esm-framework';
+import { type Order } from '@openmrs/esm-patient-common-lib';
+import classNames from 'classnames';
+import lowerCase from 'lodash/lowerCase';
+import upperCase from 'lodash/upperCase';
```

Comment:
Don't use lodash. Use `lodash-es` and import like normal. 

That said, generally using these functions is a mistake. We generally want strings to come from translations and not be forcibly cased so that we can actually support an translatable UI. Please remove these functions from the component.